app/controllers/admin_raw_email_controller.rb

```diff
@@ -21,12 +21,12 @@ def show
           # is associated with, so we can display that.
           email = @raw_email.incoming_message.from_email
           domain = PublicBody.extract_domain_from_email(email)
-
-          if domain.nil?
-            @public_bodies = []
+          @public_bodies = if domain.nil?
+            []
           else
-            @public_bodies = PublicBody.find(:all, :order => "name",
-                                             :conditions => [ "lower(request_email) like lower('%'||?||'%')", domain ])
+            PublicBody.where("lower(request_email) like lower('%'||?||'%')",
+                             domain).
+              order('name')
```

Comment:
Could make this a little clearer like:

```ruby
PublicBody.
  where("lower(request_email) like lower('%'||?||'%')", domain).
    order('name')
```

At first glance I thought `domain` was a chained method rather than an argument to `where`.